source/clear-linux/getting-started/bare-metal-install/bare-metal-install.rst

```diff
@@ -370,3 +354,6 @@ CMS-based website on your |CL| system.
 
 .. _`wheel group`:
    https://en.wikipedia.org/wiki/Wheel_(Unix_term)
+
+.. _`tutorials`:
```

Comment:
Do not hard-code links withing the documentation. This link is not needed if you use the ref role and the label `tutorials` exists, which it does since you added it.